src/test/java/com/williamfeliciano/sba_movies/services/MoviesServiceTests.java

```diff
@@ -0,0 +1,87 @@
+package com.williamfeliciano.sba_movies.services;
+
+import com.williamfeliciano.sba_movies.dtos.MovieDetailsDto;
+import com.williamfeliciano.sba_movies.dtos.MovieDto;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest
```

Comment:
This annotation is mainly used to load all the Spring context. For unit tests you don't need it.
If you try to use Mocks with Mockito, a better and faster solution would be to use the following annotation: `@ExtendWith(MockitoExtension.class)` which loads the mocks but not the entire Spring context